pkg/virt-handler/mount-manager/utils/recorder_test.go

```diff
@@ -0,0 +1,215 @@
+package recorder
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/api/equality"
+	v1 "kubevirt.io/api/core/v1"
+	"kubevirt.io/client-go/api"
+)
+
+var _ = Describe("Recorder", func() {
+	var (
+		m       MountRecorder
+		vmi     *v1.VirtualMachineInstance
+		tempDir string
+	)
+	const (
+		stateDir = "test-mount-state"
+	)
+
+	Context("From cache", func() {
+		var record *vmiMountTargetRecord
+
+		BeforeEach(func() {
+			tempDir = GinkgoT().TempDir()
+			m = NewMountRecorder(filepath.Join(tempDir, stateDir))
```

Comment:
Why do you need `stateDir` here? It seems that just `tempDir` would be sufficient for the tests, no?
```suggestion
			m = NewMountRecorder(tempDir)
```

Reply:
Done! Thanks!